CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ include(GNUInstallDirs)
 # Set when installed via make install
 set(open62541_TOOLS_DIR ${PROJECT_SOURCE_DIR}/tools)
 set(open62541_NODESET_DIR ${PROJECT_SOURCE_DIR}/deps/ua-nodeset)
+set(XDP_PATH "/usr/local/src/bpf-next")
```

Comment:
The variable is defined two times.
Also in /tests/CmakeLists.txt.

You only use the path to access the .a file.
It would be better to say `XDP_LIBRARY=/usr/local/src/bpf-next/tools/lib/bpf/libbpf.a`.
That uses existing CMake naming schemes and probably also works if bpf tooling is installed from a package.
(Where the `tools` folder may not be present.)